docs/FAQ.md

```diff
@@ -36,8 +179,7 @@ a strong assertion to correctness that can be repeatedly extended and verified.
 It therefore relies on the ability to know what correct is and to securely
 verify measurement with expected correctness.
```

Comment:
The previous section could use some grammatical and sentence structure work but it is not really part of this PR. Maybe it can be fixed up in a subsequent PR after all the work here is in.